src/components/form/Form.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { RefObject } from 'react';
+
+import './Form.css';
+import Select from './Select';
+import LabeledInput from './LabeledInput';
+import genres from '../../assets/data/genres.json';
+import CustomCard from '../card/Card';
+import { textInputIsValid, dateInputIsValid, platformInpitIsValid } from '../../utils/validation';
+import type { CardProps, FormState } from '../../utils/types';
+
+export default class Form extends React.Component<unknown, FormState> {
+  form: RefObject<HTMLFormElement> = React.createRef();
```

Comment:
Старайся выводить тип, а не описывать. Так надёжнее. Чем меньше типов ты пишешь, тем проще читать и поддерживать код.

Тут было бы неплохо

```suggestion
  form = React.createRef<HTMLFormElement>();
```